libs/gi/ui/src/components/character/StatDisplayComponent.tsx

```diff
@@ -42,28 +67,54 @@ export function StatDisplayComponent({
 
 function Section({
   displayNs,
+  compareDisplayNs,
   sectionKey,
 }: {
-  displayNs: DisplaySub<CalcResult>
+  displayNs?: DisplaySub<CalcResult>
+  compareDisplayNs?: DisplaySub<CalcResult>
   sectionKey: string
 }) {
   const optimizationTarget = useContext(OptimizationTargetContext)
   const { data, compareData } = useContext(DataContext)
   const database = useDatabase()
   const header = useMemo(
-    () => getDisplayHeader(data, sectionKey, database),
-    [database, data, sectionKey]
-  )
-  const displayNsReads = useMemo(
     () =>
-      objMap(displayNs, (n, nodeKey) =>
-        customRead(['display', sectionKey, nodeKey])
-      ),
-    [displayNs, sectionKey]
+      getDisplayHeader(data, sectionKey, database) ??
+      (compareData && getDisplayHeader(compareData, sectionKey, database)),
+    [database, data, compareData, sectionKey]
   )
-  if (!header) return <CardThemed></CardThemed>
+  const keys = useMemo(() => {
+    const keySet = new Set<string>()
+    if (displayNs) Object.keys(displayNs).map((k) => keySet.add(k))
+    if (compareDisplayNs)
+      Object.keys(compareDisplayNs).map((k) => keySet.add(k))
+    return Array.from(keySet)
+  }, [compareDisplayNs, displayNs])
+
+  if (!header) return null
+  if (!keys.length) return null
 
   const { title, icon, action } = header
+  const fields = keys.map((key) => (
+    <NodeFieldDisplay
+      key={key}
+      calcRes={displayNs?.[key]}
+      compareCalcRes={compareDisplayNs?.[key]}
+      component={ListItem}
+      emphasize={
+        JSON.stringify(optimizationTarget) ===
+        JSON.stringify([
+          sectionKey,
+          displayNs?.[key] ?? compareDisplayNs?.[key],
+        ])
+      }
+      diffOnly={sectionKey === 'character'}
+      showZero={sectionKey === 'baisc' && key === 'eleMas'}
```

Comment:
```suggestion
      showZero={sectionKey === 'basic' && key === 'eleMas'}
```

Reply:
also prob want this for:
elemental dmg% of the character
phys dmg%
healing %

I think we used to have these overrides somewhere